### PR TITLE
move js out of templates

### PR DESCRIPTION
**Pull request by adeelq**

### CODE REVIEW

File: Block/Roi.php [hunk 1]
```diff
@@ -44,8 +44,33 @@ public function isRoiTrackingEnabled()
     /**
      * @return \Magento\Sales\Model\Order
      */
-    public function getOrder()
+    private function getOrder()
```
Comment (cdiacon) [inline Block/Roi.php]: @adeelq you can't access private methods from templates

Comment (adeelq) [inline Block/Roi.php]: @cdiacon this is only used internally in the class not in template. see the refactored code.

File: view/frontend/templates/email/tracking_code.phtml [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-            var s = document.getElementsByTagName('script')[0];
-            s.parentNode.insertBefore(connector, s);
-        })();
-    </script>
-<?php endif;
+<script type="text/x-magento-init">
+    {
+        "*": {
+            "trackingCode": {
+                "isEnabled": "<?php echo (boolean) $helper->isPageTrackingEnabled(); ?>"
```
Comment (cdiacon) [inline view/frontend/templates/email/tracking_code.phtml]: isPageTrackingEnabled - is returning already only boolean type

File: view/adminhtml/web/js/preview.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+            }
+        });
+    }
+
+    /**
+     * Filter styles
+     * @param id
+     * @param value
+     */
+    function doYourThing(id, value) {
```
Comment (cdiacon) [inline view/adminhtml/web/js/preview.js]: change function name

**Pull request merged**